iocore/net/QUICNetVConnection.cc

```diff
@@ -204,28 +203,39 @@ QUICNetVConnection::free(EThread *t)
 {
   QUICConDebug("Free connection");
 
+  this->_udp_con = nullptr;
+  _unschedule_packet_write_ready();
+
+  delete this->_handshake_handler;
+  delete this->_application_map;
+  delete this->_hs_protocol;
+  delete this->_loss_detector;
+  delete this->_frame_dispatcher;
+  delete this->_stream_manager;
+  delete this->_congestion_controller;
+
+  super::clear();
```

Comment:
Since it extends multiple class, let's use their names instead of `super`.

Reply:
I think we just leave this until we can uncomment it .